uwblueprint/sistering
# Shifts table header feature

## Ticket link

Closes #123 


## Implementation description
* Use user session to call shift signups for volunteer
* Filter by status (PUBLISHED vs else)
* Filter by week/month/all using moment js


## Steps to test
1. Login as a volunteer
2. go to route /volunteer/shifts
3. Play around with different dates (same week/month/year, etc)
4. Ensure that the two tabs filter properly and date filters start time and auto close time respectively for each tab


## What should reviewers focus on?
* Correctness

## Checklist
